tests/dragonfly/replication_test.py

```diff
@@ -1838,31 +1838,62 @@ async def test_replicaof_reject_on_load(df_local_factory, df_seeder_factory):
     replica.stop()
 
 
+# note: please be careful if you want to change any of the parameters used in this test.
+# changing parameters without extensive testing may easily lead to weak testing case assertion
+# which means eviction may not get triggered.
 @pytest.mark.asyncio
 @pytest.mark.slow
 async def test_eviction_propagation(df_local_factory, df_seeder_factory):
-    master = df_local_factory.create(proactor_threads=1, cache_mode="true", maxmemory="290mb")
+    master = df_local_factory.create(
+        proactor_threads=1, cache_mode="true", maxmemory="267mb", enable_heartbeat_eviction="false"
```

Comment:
add a comment that we require 256_MB per thread in dragonfly

Reply:
actually why not set maxmemory=256

Reply:
changed to 256... it doesn't matter that much (on the timing)